Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems

```diff
@@ -1324,6 +1325,9 @@
     <Compile Update="$(MSBuildThisFileDirectory)Issue6254.xaml.cs">
       <DependentUpon>Issue6254.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\Sources\GitHub\Xamarin.Forms\Xamarin.Forms.Controls.Issues\Xamarin.Forms.Controls.Issues.Shared\Issue5108.xaml.cs">
```

Comment:
```suggestion
    <Compile Update="$(MSBuildThisFileDirectory)Issue5108.xaml.cs">
```

Reply:
To prevent any build errors on hardcoded paths there is a variable in place here. Probably not done by you on purpose since it's generated code :)